packages/fastify-api-reference/src/fastifyApiReference.ts

```diff
@@ -15,15 +18,47 @@ export type FastifyApiReferenceOptions = {
    * If you’re prefixing Fastify with a path, you can set it here.
    * It’ll be added to the JavaScript URL and the route.
    *
-   * Example: ${publicPath}${routePrefix}/@scalar/fastify-api-reference/js/browser.js
+   * Example: `${publicPath}${routePrefix}/@scalar/fastify-api-reference/js/browser.js`
    */
   publicPath?: string
   /**
    * Prefix the route with a path. This is where the API Reference will be available.
    *
    * @default '/reference'
    */
-  routePrefix?: string
+  routePrefix?: `/${string}`
+  /**
+   * Set where the OpenAPI specification is exposed under `${routePrefix}`.
+   *
+   * The specification is always available on these endpoints, parsed by `@scalar/openapi-parser`.
+   *
+   * The specification is sourced from, in order of precedence:
+   * - `configuration.spec.content`
+   * - `configuration.spec.url` – fetched via `@scalar/openapi-parser/plugins/fetch-urls`
+   * - `@fastify/swagger` – if `configuration.spec` is not provided
+   *
+   * These endpoints can be used to fetch the OpenAPI specification for your own programmatic use.
+   *
+   * @default{ json: '/json', yaml: '/yaml' }
+   */
+  openApiSpecEndpoint?: {
```

Comment:
and can we name this `openApiDocumentEndpoints` (my fav), `openApiDocuments` or `openApiFiles`?

Reply:
renamed to `openApiDocumentEndpoints` 👍